nqeit-iceberg-dremio-extension/src/main/java/org/projectnessie/integtests/dremio/DremioHelper.java

```diff
@@ -124,7 +132,19 @@ private void waitForJobCompletion(String jobId, String query) throws IOException
     Instant deadline = Instant.now().plus(timeout);
 
     while (true) {
-      String responseBody = performHttpRequest(url, null);
+      sleepSec(10); // avoid making too many api calls
```

Comment:
Wouldn't it be good enough to sleep for just 1 sec when handling the HTTP/429?
10 + 30 seconds feels very long.